app/src/main/java/com/pusher/demo/features/marketplace/ChatkitManager.kt

```diff
@@ -15,11 +16,25 @@ object ChatkitManager {
     private lateinit var chatManager: ChatManager
     var currentUser: CurrentUser? = null
 
-    interface ChatManagerConnectedListener{
+    interface ChatManagerConnectedListener {
         fun onConnected(user: CurrentUser)
         fun onError(error: String)
     }
 
+    interface ChatManagerAllUsersListener {
+        fun onUsers(users: List<User>)
```

Comment:
How about the following naming:
```kotlin
interface JoinedRoomsMembersFetchListener {
  fun onMembersFetched(users: List<User>)
```
That's a more precise naming as it's not about all users in the chat instance but just members of current user's joined rooms. As well it would be good to a verb of what the async callback is waiting for. Also `ChatManager` prefix is not necessary, indeed as it's an inner interface that prefix effectively already exists.

Reply:
Good point. I've renamed this interface `JoinedRoomsMembersListener` as the fetch is part of one of the methods.

Reply:
I also updated the `getAllUsers` method to read what it's actually doing `getUsersFromMyJoinedRooms`